src/main/resources/application.properties

```diff
@@ -1,17 +1,24 @@
+spring.config.import=optional:secrets.properties
+
+secret.properties=spring.data.mongodb.username \
+  spring.data.mongodb.password \
+  coinmarketcap.key \
+  nonexistingpropertyone \
+  nonexistingpropertytwo
+
+spring.data.mongodb.username=y/F6JmNCR2o9bREe2UgceyuK79M=
+spring.data.mongodb.password=1PF7NbY8WYDcmEPzrVWb5VU+egbr
```

Comment:
Can we keep user and password in external file on a host, so in configuration we have a link to file and read from file user name and password